packages/backend/src/open_payments/auth/middleware.ts

```diff
@@ -52,6 +56,19 @@ function contextToRequestLike(ctx: HttpSigContext): RequestLike {
   }
 }
 
+function toOpenPaymentsAccess(
+  type: AccessType,
+  action: RequestAction,
+  identifier?: string
+): AccessItem {
+  return {
+    // eslint-disable-next-line @typescript-eslint/no-explicit-any
+    type: type as any,
+    actions: [action],
+    identifier: identifier ?? undefined
+  }
+}
```

Comment:
i tried to make this fully typesafe and it got brutal 😆 

Effectively the same, but perhaps a little cleaner?

```suggestion
function toOpenPaymentsAccess(
  type: AccessType,
  action: RequestAction,
  identifier?: string
): AccessItem {
  return {
    type: type,
    actions: [action],
    identifier: identifier ?? undefined
  } as AccessItem
}
```